src/behaviour.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+from src.utils import is_list_of_type
+
+# Enum defining the possible results of classify
+Label = Enum('Label', 'Ok Undesired')
+
+class Frame:
+  def __init__(self):
+        self.id = 0
+        self.rssi = []
+        self.packet_length = 0
+        self.mac_address = 0
```

Comment:
```suggestion
        self.transmitter_mac_address = 0
```
Tænker det er både mere præcist beskrivelse og mere kompatibel med hvis vi skal have både transmitter og receiver mac address på. :)